tests/collection.tests.ts

```diff
@@ -33,20 +33,18 @@ describe('Collection tests', () => {
         }
       ]
     }
-    // const collection_name = 'new_collection_name'
-    // const update_collection_payload = {
-    //   name: collection_name,
-    //   site_name: 'test_site',
-    //   current_collection_name: 'test_collection',
-    //   external_details: {
-    //     enabled: true,
-    //     external_id: '1',
-    //     external_endpoint: '2',
-    //     page_item_url_field: '3',
-    //     collection_data_json_path: '4',
-    //     authorization_header_value: '5',
-    //   }
-    // }
+    const collection_name = 'new_collection_name'
+    const update_collection_payload = {
```

Comment:
We can remove some duplication here, I think: 

create a new const for the details:
```
const external_details = {
  enabled: true,
  external_id: '1',
  ... rest of props
}
```

Then update this const to be:

```
const update_collection_payload = {
  name: collection_name,
  external_details: external_details
}
```

This will let us reuse this down below in the test, instead of duping all this.